.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea
+venv
+pyc
```

Comment:
This one must be 
```suggestion
*.pyc
```

Reply:
You does not have folder pyc)

Reply:
Without this restriction, I will not pass the tests)